### PR TITLE
Test cases for "ew", "lt", "le", "in", "out", "w"

### PR DESCRIPTION
**Pull request by hayas**

### CODE REVIEW

File: src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java
Comment (kfrankic) [inline src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java]: Throughout these tests, use res.data() instead of res.getJson().getArray("data");

**Review by kfrankic — CHANGES_REQUESTED** [review]
Make changes!

File: src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java
Comment (kfrankic) [inline src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java]: 'json' is no longer used in this test.

File: src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java
Comment (wellsb1) [inline src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java]: Functionally good.  Would prefer this...

assertTrue(data.size() > 0);
for (Object o : data)
{
  assertTrue(((ObjectNode)o).getString("shipname").endsWith("Chevalier")));
}

File: src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java
Comment (wellsb1) [inline src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java:260]: Please check the employeeid of the returned values to make sure the do not include 1 or 2.  Can leave the 7 length check but one of those seven could be 1 or 2.

File: src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java
Comment (wellsb1) [inline src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java:290]: Please also check that the response values include "ondon".

**Review by wellsb1 — CHANGES_REQUESTED** [review]
Additinally, please add tests for these to TestSqlQuery as you did for emp and nemp.

**Comment by hayas** [comment]
Resolved all the requested changes and added tests for "lt" and "le" operators in TestSqlQuery. All other operators("ew", "in", "out", "w") are already there

**Review by wellsb1 — APPROVED** [review]

**Comment by wellsb1** [comment]
@hayas please resolve this merge conflict and merge.

**Comment by hayas** [comment]
Resolved the merge conflicts

**Review by wellsb1 — APPROVED** [review]

**Comment by wellsb1** [comment]
@kfrankic I think you requested changes here.  Can you please review?

@hayas, please go ahead and merge after you get Kevin's approval.

CC @jwwiggs

**Review by kfrankic — APPROVED** [review]

**Pull request merged**
